src/containerapp/azext_containerapp/_utils.py

```diff
@@ -148,7 +153,7 @@ def check_unique_bindings(cmd, service_connectors_def_list, service_bindings_def
         return True
 
 
-def parse_service_bindings(cmd, service_bindings_list, resource_group_name, name, customized_keys=None):
+def parse_service_bindings(cmd, service_bindings_list, resource_group_name, name, containerapp_def, customized_keys=None):
```

Comment:
There are two places using this function: `az containerapp update`, `az containerapp create`.
You should change them all.

I check in the code, the value you need is `environmentId`, should not input `containerapp_def` directly.

Reply:
Thanks a lot. I have changed it to `environment_id`. And update the 2 `parse_service_bindings` in `containerapp_decorator.py` for `create` and `update`